### ISSUE TITLE
Stagger row labels

### ISSUE THREAD

**Issue by sjackman** [renozao/NMF#48]
I have 100 rows, which makes the font size of the row labels about 5 pt. Do you have any suggestions on how to make the printed row labels legible? Perhaps they could be staggered like so, which would allow doubling the font size to 10 pt, a big improvement.

.|.
-------|-------
row1 | .
. | row2
row3 | .
. | row4

**Comment by renozao** [comment]
Do you mean that each row label would overlap on the row above?
Saving to a pdf file with increased height should help (use argument `filename="myfile.pdf"`).

**Comment by slowkow** [comment]
I think Shaun means something like this:

![nmf-48](https://cloud.githubusercontent.com/assets/209714/8934799/6a45ac98-3517-11e5-9308-65c4041bb180.png)

I've also run into the problem of tiny row labels, and my solution as of now is to turn them off when there are too many rows.

**Comment by sjackman** [comment]
Yes, precisely. Thanks, Kamil, for the helpful figure. The font size could be even larger than one box height, say perhaps 1.5 box heights.

**Comment by renozao** [comment]
The `devel` version could get what you want, although I need to fix an issue with relative component sizes and positions:

```R
x <- rmatrix(100, 10)
lab <- paste0('Row', 1:100)
i <- seq(2, 100, by = 2)
lab[i] <- paste0(paste0(rep(' ', max(nchar(lab)) * 2 + 2), collapse = ''), lab[i])
aheatmap(x, cexRow=1.5, labRow=lab, Rowv = NA)
```

**Comment by sjackman** [comment]
Thanks, Renaud! This hack seems to work with `NMF::aheatmap` version 0.20.6, except that `cexRow` doesn't seem to affect the fontsize for the rownames. Any idea why? How do I modify this script to make it work with `Rowv` = default clustering.
